### ISSUE TITLE
Highlight `console` output

### ISSUE THREAD

**Issue by stdedos** [simov/markdown-viewer#108]
> From here https://stackoverflow.com/a/52586193/2309247

I am not sure who's "responsibility" is to update; so feel free to educate me.

Maybe those errors are related?
```
Access to XMLHttpRequest at 'file:///?preventCache=1581322597365' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, https.
GET file:/// net::ERR_FAILED
```

**Comment by simov** [comment]
Not sure what your issue exactly is, are you saying that the syntax highlighting isn't working for you?

Otherwise that error is something new in Chrome 80, just updated it yesterday, and I'm seeing it for the first time. So it seems auto reloading for local files won't work anymore unless I do something about it ..

**Comment by stdedos** [comment]
Try e.g. this:

> ```
> 	```console
> 	yu@hu:~$ sudo apt install php libapache2-mod-php
> 	yu@hu:~$ sudo systemctl restart apache2
> 	yu@hu:~$ systemctl status php7.2-fpm
> 	* php7.2-fpm.service - The PHP 7.2 FastCGI Process Manager
> 	   Loaded: loaded (/lib/systemd/system/php7.2-fpm.service; enabled; vendor preset: enabled)
> 	   Active: active (running) since Sat 2018-06-30 23:56:14 PDT; 1min 28s ago
> 	     Docs: man:php-fpm7.2(8)
> 	 Main PID: 10080 (php-fpm7.2)
> 	   Status: "Processes active: 0, idle: 2, Requests: 0, slow: 0, Traffic: 0req/sec"
> 	    Tasks: 3 (limit: 2321)
> 	   CGroup: /system.slice/php7.2-fpm.service
> 	           |-10080 php-fpm: master process (/etc/php/7.2/fpm/php-fpm.conf)
> 	yu@hu:~$ sudo systemctl restart nginx
> 	yu@hu:~$ sudo apt install php-[extname]
> 	```
> ```

on a local file (I am not sure how to produce "clean" markdown; use the quoting functuion).

I think the output won't be this:
```console
yu@hu:~$ sudo apt install php libapache2-mod-php
yu@hu:~$ sudo systemctl restart apache2
yu@hu:~$ systemctl status php7.2-fpm
* php7.2-fpm.service - The PHP 7.2 FastCGI Process Manager
   Loaded: loaded (/lib/systemd/system/php7.2-fpm.service; enabled; vendor preset: enabled)
   Active: active (running) since Sat 2018-06-30 23:56:14 PDT; 1min 28s ago
     Docs: man:php-fpm7.2(8)
 Main PID: 10080 (php-fpm7.2)
   Status: "Processes active: 0, idle: 2, Requests: 0, slow: 0, Traffic: 0req/sec"
    Tasks: 3 (limit: 2321)
   CGroup: /system.slice/php7.2-fpm.service
           |-10080 php-fpm: master process (/etc/php/7.2/fpm/php-fpm.conf)
yu@hu:~$ sudo systemctl restart nginx
yu@hu:~$ sudo apt install php-[extname]
```

----

> Otherwise that error is something new in Chrome 80, just updated it yesterday, and I'm seeing it for the first time. So it seems auto reloading for local files won't work anymore unless I do something about it ..

Okay, sorry for piggy-backing. Create a new issue or "silently" fix it.

**Comment by simov** [comment]
This extension uses Prism for syntax highlighting, so unless the syntax you want is one of the [Supported Languages](https://prismjs.com/#supported-languages), then no luck. And I can't find a `console` one.

**Comment by stdedos** [comment]
It is there: It's called "Shell Session"

**Comment by simov** [comment]
Ok, I'm adding it in the next release .. have to fix the autoreload for local files anyway. Thanks for letting me know :+1:

**Comment by simov** [comment]
Published in v3.9, also the autoreload for file URLs was fixed as well.
